tests/DefaultHttpAdapterTest.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Http\Adapter\Guzzle6\Tests;
+
+use Http\Adapter\Guzzle6\Client;
+use Http\Client\Tests\HttpClientTest;
+
+/**
+ * @author GeLo <geloen.eric@gmail.com>
```

Comment:
We don't know who to blame when something goes wrong. :stuck_out_tongue: Please add yourself @dbu